config/clients/go/template/client/client.mustache

```diff
@@ -416,6 +416,19 @@ func (client *{{appShortName}}Client) getAuthorizationModelId(authorizationModel
 	return modelId, nil
 }
 
+// helper function to validate the connection (i.e., get token)
+func (client *{{appShortName}}Client) validateAuthorizationModel(ctx _context.Context, authorizationModelId *string) error {
```

Comment:
Can you call this `checkValidApiConnection`? The intention is to make sure we can talk to the API, not read the model.

Also to be consistent with the other SDKs

Reply:
updated